spec/pluralrules.html

```diff
@@ -146,23 +136,19 @@
         1. Assert: Type(_n_) is Number.
         1. If isFinite(_n_) is *false*, then
           1. Return *"other"*.
-        1. If the _pluralRules_.[[MinimumSignificantDigits]] is not *undefined* and _pluralRules_.[[MaximumSignificantDigits]] is not *undefined*, then
-          1. Let _s_ be ToRawPrecision(_n_, _pluralRules_.[[MinimumSignificantDigits]], _pluralRules_.[[MaximumSignificantDigits]]).
-        1. Else,
-          1. Let _s_ be ToRawFixed(_n_, _pluralRules_.[[MinimumIntegerDigits]], _pluralRules_.[[MinimumFractionDigits]], _pluralRules_.[[MaximumFractionDigits]]).
         1. Let _locale_ be _pluralRules_.[[Locale]].
         1. Let _type_ be _pluralRules_.[[Type]].
+        1. Let _nfOptions_ be ObjectCreate(*%ObjectPrototype%*).
+        1. Set _nfOptions_[*"minimumIntegerDigits"*] to _pluralRules_.[[MinimumIntegerDigits]].
+        1. Set _nfOptions_[*"minimumFractionDigits"*] to _pluralRules_.[[MinimumFractionDigits]].
+        1. Set _nfOptions_[*"maximumFractionDigits"*] to _pluralRules_.[[MaximumFractionDigits]].
+        1. Set _nfOptions_[*"minimumSignificantDigits"*] to _pluralRules_.[[MinimumSignificantDigits]].
+        1. Set _nfOptions_[*"maximumSignificantDigits"*] to _pluralRules_.[[MaximumSignificantDigits]].
```

Comment:
Right, and on top of that, why are we creating a new obj? I thought https://github.com/tc39/ecma402/pull/104 was suppose to provide a way for _nf_ and _pf_ and any other format to be passed as the first argument, so there is not need to create this object every time.